src/components/OSCALSystemImplementation.js

```diff
@@ -31,6 +31,140 @@ const OSCALSystemImplementationSubDataHeader = styled(TableCell)`
+                  component="th"
+                  scope="row"
+                >
+                  {property.name}
+                </OSCALSystemImplementationSubDataHeader>
+                <SmallTableCell>{property.value}</SmallTableCell>
+              </TableRow>
+            ))}
+        </TableBody>
+      </Table>
+    </TableContainer>
+  );
+}
+
+function OSCALSystemImplementationComponents(props) {
+  return (
+    <>
+      <OSCALSystemImplementationTableTitle variant="h6" id="tableTitle">
+        Components
+      </OSCALSystemImplementationTableTitle>
+      <TableContainer>
+        <Table aria-label="Components">
+          <TableHead>
+            <TableRow>
+              <TableCell>Component</TableCell>
+              <TableCell>Type</TableCell>
+              <TableCell>Status</TableCell>
+              <TableCell>Properties</TableCell>
+              <TableCell>Responsible Roles</TableCell>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {Object.entries(props.components).map(([key, component]) => (
+              <TableRow key={key}>
+                <ComponentTableCell component="th" scope="row">
+                  <StyledTooltip title={component.description}>
+                    <Typography variant="body2">{component.title}</Typography>
+                  </StyledTooltip>
+                </ComponentTableCell>
+                <TableCell>{component.type}</TableCell>
+                <TableCell>
+                  {component.status && component.status.state}
+                </TableCell>
+                <TableCell>
+                  <PropertiesTable list={component.props} />
+                </TableCell>
+                <TableCell>
+                  <OSCALResponsibleRoles
+                    responsibleRoles={component["responsible-roles"]}
+                    parties={props.parties}
+                  />
+                </TableCell>
+              </TableRow>
+            ))}
+          </TableBody>
+        </Table>
+      </TableContainer>
+    </>
+  );
+}
+
+function AuthorizedPrivilegesTable(props) {
+  return (
+    <TableContainer>
+      <Table size="small">
+        <TableBody>
+          {props.authorizedPrivileges?.map((privilege) => (
+            <TableRow key={privilege.title}>
+              <TableCell>{privilege.title}</TableCell>
+            </TableRow>
+          ))}
+        </TableBody>
+      </Table>
+    </TableContainer>
+  );
+}
+
+function OSCALSystemImplementationUsers(props) {
+  return (
+    <>
+      <OSCALSystemImplementationTableTitle variant="h6" id="tableTitle">
+        Users
+      </OSCALSystemImplementationTableTitle>
+      <TableContainer>
+        <Table aria-label="Components">
+          <TableHead>
+            <TableRow>
+              <TableCell>Title</TableCell>
+              <TableCell>Short Name</TableCell>
+              <TableCell>Properties</TableCell>
+              <TableCell>Authorized Privileges</TableCell>
```

Comment:
I felt that this would be good information to display. Is there any other fields we may want to display? 

@brian-easyd

Reply:
In general, a good sanity check on what to display, always consider looking at the [FedRAMP System Security Plan Template](https://www.fedramp.gov/assets/resources/templates/FedRAMP-SSP-High-Baseline-Template.docx). In this case, _Section 9.3, Types of Users_.

![image](https://user-images.githubusercontent.com/106116211/187275253-599ceb11-6170-4b4d-ad37-73ec7107731a.png)

Although this is a rare case where the FedRAMP example uses an OSCAL extension, as "Sensitivity" is not part of the core OSCAL syntax. Otherwise, this is a good reference.